agents/telegraf.go

```diff
@@ -274,3 +275,75 @@ func (tr *TelegrafRunner) hasRequiredPaths() bool {
 
 	return true
 }
+
+func (tr *TelegrafRunner) ProcessTestMonitor(correlationId string, content string) (*telemetry_edge.TestMonitorResults, error) {
+
+	// Convert content to TOML
+
+	configToml, err := ConvertJsonToTelegrafToml(content, nil)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to convert config content")
+	}
+
+	// Generate token/id used for authenticating and pulling telegraf config
+
+	testConfigServerToken := uuid.NewV4().String()
+	testConfigServerId := uuid.NewV4().String()
+
+	// Bind to the next available port by using :0
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	if err != nil {
+		return nil, errors.Wrap(err, "couldn't create http listener")
+	}
+	//noinspection GoUnhandledErrorResult
+	defer listener.Close()
+
+	listenerPort := listener.Addr().(*net.TCPAddr).Port
+	hostPort := fmt.Sprintf("127.0.0.1:%d", listenerPort)
+
+	configServerErrors := make(chan error, 2)
+
+	testConfigRunner := telegrafTestConfigRunnerBuilder(testConfigServerId, testConfigServerToken)
+
+	// Start the config server
+
+	configServer := testConfigRunner.StartTestConfigServer(configToml, configServerErrors, listener)
+
+	// Rung the telegraf test command
+
+	results := &telemetry_edge.TestMonitorResults{
+		CorrelationId: correlationId,
+		Errors:        []string{},
+	}
+	cmdOut, err := testConfigRunner.RunCommand(hostPort, tr.exePath(), tr.basePath)
+	if err != nil {
+		results.Errors = append(results.Errors, "Command: "+err.Error())
```

Comment:
Since you've been fixing other things, should this not be
```suggestion
		results.Errors = append(results.Errors, "Command: " + err.Error())
```

Reply:
Looks like there could be a few others like that.

Reply:
Go fmt actually changed the others to eliminate that space (such as [this spot](https://github.com/racker/salus-telemetry-envoy/pull/55/files#diff-53149eb7758cacde61602dc5daea59dfR97) that got changed automatically during this PR). I believe Google's Java standard likes the space, which adds to the confusion.